actions/run_action.go

```diff
@@ -1,3 +1,33 @@
+/*
+Run Action
+
+Allows to run any available command or script in target chroot or
+in host environment.
+
+Yaml syntax:
+ - action: run
+   chroot: bool
+   postprocess: bool
+   script: script name
+   command: command line
+
+Mandatory properties:
+
+- command -- command with arguments; the command expected to be accessible in
+host's or chrooted environment -- depending on 'chroot' property.
+
+- script -- shell script with arguments; script must be provided.
```

Comment:
script doesn't have to be a shell script though it would typically be. you'd also want to note that the script path is relative to the recipe directory